Gemfile

```diff
@@ -34,19 +33,20 @@ gem 'sidekiq'
 
 # Services gems
 gem 'minigit', '~> 0.0.4'
-gem 'net-ssh', '~> 7.3.0'
+gem 'net-ssh', '~> 7.3'
 
 gem 'awesome_spawn',        '~> 1.6'
 gem 'default_value_for',    '~> 4.0'
-gem 'haml_lint',            '~> 0.51', :require => false
+gem 'haml_lint',            '~> 0.52', :require => false
 gem 'irb'
-gem 'manageiq-style',       '~> 1.5', ">=1.5.6", :require => false
+gem 'manageiq-style',       '~> 1.5.6',  :require => false
 gem 'more_core_extensions', '~> 4.4',  :require => 'more_core_extensions/all'
 gem 'rugged',                          :require => false
 
 gem 'faraday'
-gem 'faraday-http-cache', '~> 2.5.0'
-gem 'octokit', '~> 4.25.0', :require => false
+gem 'faraday-http-cache', '~> 2.5'
```

Comment:
same thing here. do we want 2.5.x or 2.x?

Reply:
ok. this looks fine for me.

looks like a slow moving gem and is maintained by someone who looks seasoned and has a lot of gems.